gateway/http.d/apicast.conf.liquid

```diff
@@ -4,6 +4,15 @@ map $status $extended_access_log {
     default '';
 }
 
+map $status $access_logs_enabled {
```

Comment:
Maps works like switch..case selectors to set values to conditional variables in nginx, if I am not mistaken. If I understand correctly then, when `default` is provided alone, the value of the variable will be the default value (when even `default` is omitted, it will be the empty string, by the way).

So why using `$status`? Out of curiosity, would something like the following work instead?

```suggestion
map "" $access_logs_enabled {
```